.vscode/settings.json

```diff
@@ -1,37 +1,44 @@
 // Cesium project settings for VSCode.
 {
-    "files.exclude": {
-        "**/.git": true,
-        "**/.DS_Store": true,
-        "**/Thumbs.db": true,
-        ".idea": true,
-        ".metadata": true,
-        ".settings": true,
-        ".externalToolBuilders": true,
-        ".project": true,
-        "launches": true
-    },
-    "search.exclude": {
-        "Build": true,
-        "node_modules": true,
-        "Source/Shaders/**/*.js": true
-    },
-    "files.associations": {
-        "*.czml": "json"
-    },
-    "files.trimTrailingWhitespace": true,
-    "files.insertFinalNewline": true,
-    "editor.insertSpaces": true,
-    "editor.tabSize": 2,
-    "editor.detectIndentation": false,
-    "eslint.enable": true,
-    "javascript.format.insertSpaceAfterCommaDelimiter": true,
-    "javascript.format.insertSpaceAfterSemicolonInForStatements": true,
-    "javascript.format.insertSpaceBeforeAndAfterBinaryOperators": true,
-    "javascript.format.insertSpaceAfterKeywordsInControlFlowStatements": true,
-    "javascript.format.insertSpaceAfterFunctionKeywordForAnonymousFunctions": false,
-    "javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyParenthesis": false,
-    "javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBrackets": false,
-    "javascript.format.placeOpenBraceOnNewLineForFunctions": false,
-    "javascript.format.placeOpenBraceOnNewLineForControlBlocks": false
+  "files.exclude": {
+    "**/.git": true,
+    "**/.DS_Store": true,
+    "**/Thumbs.db": true,
+    ".idea": true,
+    ".metadata": true,
+    ".settings": true,
+    ".externalToolBuilders": true,
+    ".project": true,
+    "launches": true
+  },
+  "search.exclude": {
+    "Build": true,
+    "node_modules": true,
+    "Source/Shaders/**/*.js": true
+  },
+  "files.associations": {
+    "*.czml": "json"
+  },
+  "files.trimTrailingWhitespace": true,
+  "[markdown]": {
+    "files.trimTrailingWhitespace": false
+  },
+  "files.insertFinalNewline": true,
+  "editor.insertSpaces": true,
+  "editor.tabSize": 2,
+  "editor.detectIndentation": false,
+  "eslint.enable": true,
+  "javascript.format.insertSpaceAfterCommaDelimiter": true,
+  "javascript.format.insertSpaceAfterSemicolonInForStatements": true,
+  "javascript.format.insertSpaceBeforeAndAfterBinaryOperators": true,
+  "javascript.format.insertSpaceAfterKeywordsInControlFlowStatements": true,
+  "javascript.format.insertSpaceAfterFunctionKeywordForAnonymousFunctions": false,
+  "javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyParenthesis": false,
+  "javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBrackets": false,
+  "javascript.format.placeOpenBraceOnNewLineForFunctions": false,
+  "javascript.format.placeOpenBraceOnNewLineForControlBlocks": false,
+  "glTF.defaultV2Engine": "Cesium",
+  "[glsl]": {
+    "editor.tabSize": 4
```

Comment:
Is there a reason we changed this and markdown here instead of in the .editorconfig (which would work for visual studio and other editors as well?)

Reply:
I think VSCode was getting confused by having formatting settings both here and in .editorconfig.  I've force-pushed my branch with an update that moves all of these settings to be exclusively .editorconfig settings, and VSCode feels more consistent now.